test-build.sh

```diff
@@ -74,9 +74,6 @@ for version in "${versions[@]}"; do
   tag=$(get_tag "${version}")
   full_version=$(get_full_version "${version}")
 
-  build "${version}" "default" "${tag}"
-  test_image "${full_version}" "default" "${tag}"
```

Comment:
what is this fix for?

Reply:
My Chakracore fix in the other PR was incorrect in removing just the `if` and leaving the contents. It should have removed these lines, so when I pushed the original PR, the build failed on the tests.

Reply:
It's actually a little clearer on my GH Actions PR https://github.com/nodejs/docker-node/pull/1194 where all the builds failed before doing the same fix

Reply:
Let's separate the fix to another PR, it has nothing to do with the version update itself.